Std/Data/Rat/Lemmas.lean

```diff
@@ -16,6 +16,8 @@ namespace Rat
 @[simp] theorem one_num : (1 : Rat).num = 1 := rfl
 @[simp] theorem one_den : (1 : Rat).den = 1 := rfl
 
+@[simp] theorem mk'_one {r : Int} : mk' r 1 Nat.one_ne_zero (Nat.coprime_one_right _) = r := rfl
```

Comment:
Other lemmas don't use the prime. It's still an odd name. Can anyone think of something better?
```suggestion
@[simp] theorem mk_one {r : Int} : ⟨r, 1, Nat.one_ne_zero, Nat.coprime_one_right _⟩ = r := rfl
```

Reply:
Would `mk_den_one` be a better name?

Reply:
This needs `(r : Rat)` on the rhs for the anonymous constructor to resolve the type.